### PR TITLE
Update values.yaml for installing openebs-monitoring with persistence storage

### PR DESCRIPTION
**Pull request by rajaSahil**
Signed-off-by: Sahil Raja <sahil.raja@mayadata.io>

- Added values in values.yaml for openebs-monitoring with persistence.
- Updated the readme for the same.
- Reduced length for truncating the string in _helpers.tpl.

### CODE REVIEW

File: deploy/charts/openebs-monitoring/README.md [hunk 1]
```diff
@@ -47,9 +45,9 @@ _See [helm install](https://helm.sh/docs/helm/helm_install/) for command documen
 
 By default this chart installs additional, dependent charts:
 
-| Repository | Name | Version |
-|------------|------|---------|
-| https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack | kube-prometheus-stack | 15.4.* |
+| Repository                                                                                 | Name                  | Version |
+| ------------------------------------------------------------------------------------------ | --------------------- | ------- |
+| https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack | kube-prometheus-stack | 15.4.\* |
```
Comment (Ab-hishek) [inline deploy/charts/openebs-monitoring/README.md]: Is the version written as `15.4.\*` correct?

File: deploy/charts/openebs-monitoring/values.yaml [hunk 2]
```diff
@@ -34,6 +34,61 @@ kube-prometheus-stack:
...
+        #     name: alert
+        #   spec:
+        #     storageClassName: cstorsc-ooqun
+        #     accessModes: ["ReadWriteOnce"]
+        #     resources:
+        #       requests:
+        #         storage: 10Gi
+        #   selector: {}
+
+  prometheusOperator:
```
Comment (Ab-hishek) [inline deploy/charts/openebs-monitoring/values.yaml:84]: Is this necessary?

Comment (rajaSahil) [reply deploy/charts/openebs-monitoring/values.yaml:84]: Yes, we can have this as this is a dependent chart.

Comment (Ab-hishek) [reply deploy/charts/openebs-monitoring/values.yaml:84]: I meant what's the significance of placing it in our `values.yaml`. Is this updated, modified or used somewhere in our chart?

File: deploy/charts/openebs-monitoring/README.md (hunk 1 above)
Comment (rajaSahil) [reply deploy/charts/openebs-monitoring/README.md]: Oh, I think it is a keyboard error 😛 . WIll fix it.

File: deploy/charts/openebs-monitoring/values.yaml (hunk 2 above)
Comment (rajaSahil) [reply deploy/charts/openebs-monitoring/values.yaml:84]: Yes, I think we should put the dependent chart enabled field in values.yaml. Although while using persistence storage `tls` has to be enabled. That's why, I have added it.

Comment (Ab-hishek) [reply deploy/charts/openebs-monitoring/values.yaml:84]: Okay.

File: deploy/charts/openebs-monitoring/README.md (hunk 1 above)
Comment (rajaSahil) [reply deploy/charts/openebs-monitoring/README.md]: Checked it. While formatting in vs-code,  `\` was being added before `*`. So made the changes to `15.4.*` .

**Review by Ab-hishek — APPROVED** [review]
LGTM.

**Pull request merged**